src/Extensions/Logging/Copy/Log.cs

```diff
@@ -0,0 +1,280 @@
+﻿/*
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
>( [](start = 13, length = 1)

what is the purpose of this copy?

Reply:
I'll remove it before completing the PR I'm using it to make sure that I'm keeping behavior of logger as close to existing as possible